.hadolint.yaml

```diff
@@ -0,0 +1,2 @@
+ignored:
+  - DL3018
```

Comment:
Do you mind adding this rule as inline in the Dockerfile itself?

Reply:
Sure, I forgot that it was possible

Reply:
done